packages/client/components/MeetingCardOptionsMenu.tsx

```diff
@@ -84,6 +86,11 @@ const MeetingCardOptionsMenu = (props: Props) => {
           closePortal()
           const copyUrl = getMassInvitationUrl(token)
           await navigator.clipboard.writeText(copyUrl)
+
+          SendClientSegmentEventMutation(atmosphere, 'Copied Invite Link', {
+            teamId: team?.id,
```

Comment:
+1: could be useful to add another derived property `inviteTo` as in [`Invite Email Sent`](https://github.com/ParabolInc/parabol/blob/3d74dd5b3037fab08b6a49d3c30d4a36f333d582/packages/server/utils/analytics/analytics.ts#L250)?

Reply:
Also, following Segment's [naming convention](https://segment.com/academy/collecting-data/naming-conventions-for-clean-data/), it should probably be called `Invite Link Copied` 🤷‍♂️

Reply:
Updated naming, but `inviteTo` isn't an allowed property on `SegmentEventTrackOptions`, and it seems incorrect to just add this property to the list.